content/logs/_index.md

```diff
@@ -271,6 +275,29 @@ logs:
         pattern: \d{4}\-(0?[1-9]|1[012])\-(0?[1-9]|[12][0-9]|3[01])
 ```
 
+### Wildcards
+
+If your log files are labeled by date or simply all stored in the same directory, configure your Datadog Agent to monitor them all and automatically detect new ones by using wildcards in the `path` attribute.
```

Comment:
we call it Datadog Agent here but Log Agent on line 155

Reply:
How does using a wildcard impact file handle usage?

Reply:
We should avoid words like `simply` in docs.

Reply:
Without wildcards you have to hardcode your log file path in your configuration file. But with log rotates or other logging strategies, it can be cumbersome to sustain.